feat(curriculum): create design fundamentals quiz #56181 (#56546)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/quiz-design-fundamentals/66ed901ff45ce3ece4053eba.md

#### --text--

> What are some reasons why whitespace is important for good design?

Reviewer: ```suggestion
What are some reasons whitespace is important in good design?
```

#### --text--

In the examples below, which one is an example of visual hierarchy in design?

#### --distractors--

Headline and title text are a similar color to the background and other elements to blend in

---

Text and images are all the same size on the page

---

The logo banner is smaller than all the minor images on the main landing page

#### --answer--

Making the headline of the page larger than other text that comprise the body or sub-headings

### --question--

#### --text--

Which Figma tools allows users to specify a region on the screen they would like to export?

#### --distractors--

Frames

---

Section

---

Blocks

#### --answer--

Slice

### --question--

#### --text--

What is user centered design?

#### --distractors--

A design process where once the design is completed, designers test the product by getting user feedback

---

A design process where designers consider user accessibility requirements

---

A design process focused on client requirements at every stage of the process

#### --answer--

An iterative design process where designers design a product considering user requirements at every stage of the process

### --question--

#### --text--

> What is best practice contrast ratio for regular text for accessibility according to WCAG guidelines?

Reviewer 2: ```suggestion
What is the minimum required contrast ratio for regular text for accessibility according to WCAG guidelines?
```

WCAG defines minimum requirements, not necessarily best practices.
